app/presenters/hyrax/admin/user_activity_presenter.rb

```diff
@@ -2,28 +2,24 @@
 module Hyrax
   module Admin
     class UserActivityPresenter
-      def initialize
-        @x_min = 90.days.ago.beginning_of_day
-        @date_format = ->(x) { x.strftime('%b %-d') }
+      def initialize(start_date, end_date)
```

Comment:
this method signature change is breaking, but could be made compatible fairly easily by providing default parameters.

```suggestion
      def initialize(start_date: 90.days.ago.beginning_of_day, end_date: Date.today.end_of_day)
```

i think i prefer the named parameters over order dependent parameters in general, but the following is fine too:

```ruby
      def initialize(start_date = 90.days.ago.beginning_of_day, end_date = Date.today.end_of_day)
````

Reply:
what does it break?   If I change to the suggested the dashboard does not load.

Reply:
@sara-g it breaks the method signature for `Hyrax::Admin::UserActivityPresenter.new`. any client code calling `UserActivityPresenter.new` without arguments will get an error about the now required arguments.

we generally try to avoid breaking method signatures like this in Hyrax, because our classes and methods can be used downstream in ways that aren't predictable to us. when we do make changes like this, we bump the major version; so this change would put us into 4.0.0 territory. since i didn't see many other changes that would require that, it seemed worth considering reinstating compatibility.

>  If I change to the suggested the dashboard does not load.

is this just because all the existing callers to `.new` have changed in this PR? i think you'd need to either use the unnamed parameters in the second suggestion, or update the callers to be inline with the proposed named parameter changes.

or is there something bigger going on?